server/lib/adapters/filesystem.js

```diff
@@ -6,7 +6,8 @@ import fs from "fs";
 import JSONStream from "JSONStream";
```

Comment:
I would move this module under the `test` folder as it's really only there for testing purposes... thoughts ?